picturae_importer.py

```diff
@@ -865,8 +850,7 @@ def upload_records(self):
             if self.taxon_id is None:
                 self.create_taxon()
 
```

Comment:
now creating new locality ID for every new collection object, to allow for smoother updating when notes from nature comes back.